### PR TITLE
Missing continue in time constraint get_data

### PR DESCRIPTION
**Pull request by sourabhdalvi**

### CODE REVIEW

File: test/test_device_thermal_generation_constructors.jl [hunk 1]
```diff
@@ -361,7 +361,7 @@ end
             use_parameters = p,
         )
         construct_device!(op_problem, :Thermal, model)
-        moi_tests(op_problem, p, 528, 0, 192, 108, 192, true)
+        moi_tests(op_problem, p, 528, 0, 188, 108, 192, true)
```
Comment (jd-lara) [inline test/test_device_thermal_generation_constructors.jl:364]: Can you comment on why this change, please?

**Comment by jd-lara** [comment]
@sourabhdalvi it seems a test is still failing.

File: test/test_device_thermal_generation_constructors.jl (hunk 1 above)
Comment (sourabhdalvi) [reply test/test_device_thermal_generation_constructors.jl]: When parameters are not used, the pglib constraint function creates spares constraint  and when the continue is added back I guess its skipping one device so drop in the constraint count
